feature/experimental/isis/otg_tests/isis_interface_hello_padding_enable_test/isis_interface_hello_padding_enable_test.go

```diff
@@ -73,7 +73,6 @@ func configureISIS(t *testing.T, ts *session.TestSession) {
 
 	// Level configs.
 	level := isis.GetOrCreateLevel(2)
-	level.Enabled = ygot.Bool(true)
```

Comment:
Why remove this? Please add the same deviation `isis_level_enabled` here.

Reply:
This script uses session.go for base ISIS config and session.go has the deviation updated for level enabled. Additional config required for this specific testcase is being added in this test file apart from the config added in session.go. Since setting level enabled is already part of session.go, removing the duplicate level enabled config from this test file.